model/task/task.go

```diff
@@ -1220,8 +1220,14 @@ func (t *Task) MarkEnd(finishTime time.Time, detail *apimodels.TaskEndDetail) er
 
 func (t *Task) GetDisplayStatus() string {
 	if t.DisplayStatus != "" {
+		if t.Aborted {
+			return evergreen.TaskAborted
+		}
 		return t.DisplayStatus
 	}
+	if t.Aborted {
```

Comment:
It's possible to request a task abort, but it hasn't quite finished yet so that it's still running. You should also check IsFinishedTaskStatus here